client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -314,6 +314,10 @@
             link: function(scope) {
                 scope.limits = authoring.limits;
                 scope._editable = scope.origItem._editable;
+                scope._isInProductionStates = scope.origItem.state !== 'published' &&
```

Comment:
can we use this 
https://github.com/akintolga/superdesk/blob/updates/client/app/scripts/superdesk-authoring/authoring.js#L273